python/BioSimSpace/Sandpit/Exscientia/_SireWrappers/_molecule.py

```diff
@@ -1879,6 +1881,20 @@ def _getPerturbationIndices(self):
 
         return idxs
 
+    def getCOMIdx(self):
+        """Get the index of the atom that closest to the center of mass."""
+        if self.isPerturbable():
+            property_map = {"coordinates": "coordinates0", "mass": "mass0"}
+        else:
+            property_map = {"coordinates": "coordinates", "mass": "mass"}
+        coords = self.coordinates(property_map=property_map)
```

Comment:
I think there was a `sire` function which gives you the numpy array of coordinates